monitoring/dashboard/values.yaml

```diff
@@ -0,0 +1,17 @@
+image:
```

Comment:
This should be in a different PR, right?

Reply:
sure